src/app/hm-admin/services/api/api.service.ts

```diff
@@ -56,8 +59,6 @@ export class APIService {
       throw Error(`Can't find an entrypoint for ${modelName} in ${this.entrypoints}`);
     }
 
-    return this.coreDefinitionService.getDefinitions().mergeMap((definition) => {
-      return this.getCollectionByUrl(this.config.api.baseUrl + entrypoint.url, definition);
-    });
+    return this.getCollectionByUrl(this.config.get('api.baseUrl') + entrypoint.url);
```

Comment:
What is bc break ? You can still use the Old way. It's juste better with this function